pptx/oxml/text.py

```diff
@@ -324,8 +326,10 @@ class CT_TextField(BaseOxmlElement):
     <a:fld> field element, for either a slide number or date field
     """
 
+    id = RequiredAttribute("id", XsdString)
```

Comment:
id is a python built-in function. Best to use a different name.

Reply:
I recognize that we are shadowing the id built-in, but since we are inside of a class, and are unlikely to need the id function here, I think the readability makes it worth it. There is also some precedent in the codebase for this: the CT_SlideId class in [pptx/oxml/presentation.py](https://github.com/scanny/python-pptx/blob/d043334b984736a7a2ade3fb6f9adcdd97b3e8f5/pptx/oxml/presentation.py#L39), the CT_Connection class in [pptx/oxml/shapes/connector.py](https://github.com/scanny/python-pptx/blob/d043334b984736a7a2ade3fb6f9adcdd97b3e8f5/pptx/oxml/shapes/connector.py#L22), and CT_NonVisualDrawingProps in [pptx/oxml/shapes/shared.py](https://github.com/scanny/python-pptx/blob/d043334b984736a7a2ade3fb6f9adcdd97b3e8f5/pptx/oxml/shapes/shared.py#L305) all have attributes named id.